osmaxx/excerptexport/models/excerpt.py

```diff
@@ -84,6 +85,13 @@ def color(self):
     def extent(self):
         return self.bounding_geometry.extent
 
+    @property
+    def has_running_exports(self):
+        return any([not i.can_be_deleted for es in self.extraction_orders.all() for i in es.exports.all()])
```

Comment:
You can use a generator expression instead of the list comprehension inside `any`. (I.e., leave out the `[` and `]`)